packages/compass-e2e-tests/package.json

```diff
@@ -17,8 +17,12 @@
     "test-packaged": "npm run test -- -- --test-packaged-app",
     "test-packaged-ci": "npm run test-ci -- -- --test-packaged-app",
     "reformat": "npm run eslint . -- --fix && npm run prettier -- --write .",
-    "start-server": "mongodb-runner start --id=e2e --topology=replset --secondaries=0 -- --port 27091",
-    "stop-server": "mongodb-runner stop --id=e2e",
+    "start-server-1": "mongodb-runner start --id=e2e-1 --topology=replset --secondaries=0 -- --port 27091",
+    "stop-server-1": "mongodb-runner stop --id=e2e-1",
+    "start-server-2": "mongodb-runner start --id=e2e-2 --topology=replset --secondaries=0 -- --port 27092",
+    "stop-server-2": "mongodb-runner stop --id=e2e-2",
+    "start-servers": "npm run start-server-1 && npm run start-server-2",
```

Comment:
There are two now. Yes we could have just had two connections to the same server, but this way we can hopefully pick up if something unexpectedly appears in the wrong server, for example.